src/main/java/duke/commands/FindCommand.java

```diff
@@ -0,0 +1,31 @@
+package duke.commands;
+
+import java.util.ArrayList;
+
+import duke.data.TaskList;
+import duke.storage.Storage;
+import duke.tasks.Task;
+import duke.ui.Ui;
+
+/**
+ * This class represents a command to find tasks that contains all
+ * the keywords provided
+ */
+public class FindCommand extends Command {
+    private String[] keyword;
```

Comment:
Since this is an array of strings maybe it could be named `keywords` making it plural?